### PR TITLE
Revolving pool UI components

### PR DESCRIPTION
**Pull request by offerijns**

### CODE REVIEW

File: config.ts [hunk 1]
```diff
@@ -18,6 +18,7 @@ interface PoolI {
     label: string
     link?: string
   }[]
+  version: 2 | 3
```
Comment (philipstanislaus) [inline config.ts:21]: Can we add the version to the validation with yup as well?

File: containers/Investment/RevolvingPool/View/TrancheOverview.tsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+import OrderCard from './OrderCard'
+import CollectCard from './CollectCard'
+import { TokenLogo } from './styles'
+
+interface Props {
+  pool: Pool
+  tranche: 'senior' | 'junior'
+}
+
+export type Card = 'home' | 'collect' | 'order' | 'invest' | 'redeem' | 'order'
```
Comment (philipstanislaus) [inline containers/Investment/RevolvingPool/View/TrancheOverview.tsx:16]:
```suggestion
export type Card = 'home' | 'collect' | 'order' | 'invest' | 'redeem'
```

**Review by philipstanislaus — APPROVED** [review]
Nice, thanks!

File: config.ts (hunk 1 above)
Comment (offerijns) [reply config.ts:21]: Good idea 👍

Comment (offerijns) [reply config.ts:21]: I'll add this in the integration PR, since it also requires the pool-config update.

**Pull request merged**